Week2/10_program.js

```diff
@@ -0,0 +1,4 @@
+let myArray =["My age","30"]; // With array is possibale to store different data, within a single variable
+console.log(myArray);
+let maxNumber = Math.pow(10, 1000);
```

Comment:
The assignment is as follows:

>  Can you compare infinities? (Not in Eyad's world) - does 6/0 === 10/0? How can you test this?

I don't think this is the right way to `compare` infinities. Do you understand what the assignment is? Or should I try to explain it differently?